x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -404,6 +407,32 @@ describe('when on the hosts page', () => {
       ).not.toBeNull();
     });
 
+    it('should include the link to reassignment in Ingest', async () => {
+      coreStart.application.getUrlForApp.mockReturnValue('/app/ingestManager');
+      const renderResult = render();
+      const linkToLogs = await renderResult.findByTestId('hostDetailsLinkToIngest');
```

Comment:
probs want to rename this const to `linkToIngest` or something

Reply:
you are correct, updated to have a name that makes sense